docs/index.asciidoc

```diff
@@ -30,8 +30,8 @@ Other S3 compatible storage solutions are not supported.
 S3 outputs create temporary files into the OS' temporary directory. 
 You can specify where to save them using the `temporary_directory` option.
 
-IMPORTANT: For configurations containing multiple s3 outputs with the restore
-option enabled, each output should define its own 'temporary_directory'.
+IMPORTANT: For configurations containing multiple s3 outputs with the `restore`
+option enabled, each output should define its own `temporary_directory`.
```

Comment:
Given the dangers, we should probably stress here that temporary directories must not be sub-folders of other outputs temporary directories otherwise data loss could occur